torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2205,6 +2205,23 @@ def sample_inputs_max_min_binary(op_info, device, dtype, requires_grad, **kwargs
                   for input_tensor, other_tensor in args_for_binary_op)
     return inputs
 
+def sample_inputs_adaptive_avg_pool2d(op_info, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+
+    # Ordered as (input shape, output size)
+    cases = (
+        ((1, 8, 8, 8), (5, 7)),
+        ((2, 8, 8, 8), (None, 7)),
+        ((1, 8, 4, 3), (5, None)),
+        ((1, 8, 4, 3), (None, None)),
```

Comment:
output_size is allowed to be a single integer according to https://pytorch.org/docs/stable/generated/torch.nn.functional.adaptive_avg_pool2d.html?highlight=adaptive_avg_pool#torch.nn.functional.adaptive_avg_pool2d, could we test that case for completeness?

Reply:
Thanks for taking a look, @zou3519 - appreciate the suggestion. I've added a case in the latest commit, and have also merged with upstream. :)